mappings/net/minecraft/client/world/WorldClient.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS cxj net/minecraft/client/world/WorldClient
+	FIELD D ticksSinceLightingClient I
```

Comment:
any reason for the client suffix? its already in WorldClient?

Reply:
I think there might be a separate server-side counter, but not sure.